docs/components/modeler/using-web-and-desktop-modeler-together.md

```diff
@@ -0,0 +1,80 @@
+---
+id: using-web-and-desktop-modeler-together
+title: Using Web and Desktop Modeler together
+description: "Understand the implications of using Web Modeler and Desktop Modeler for modeling process diagrams."
+---
+
+If your team uses both [Web Modeler](/components/modeler/web-modeler/launch-web-modeler.md) and [Desktop Modeler](/components/modeler/desktop-modeler/index.md) to develop [process applications](/components/concepts/process-applications.md) with [Git sync](/components/modeler/web-modeler/git-sync.md), there are a few considerations to ensure both modelers work together transparently.
```

Comment:
🔧. I don't know about the "If" here as an introduction. The goal of this page should be to introduce how to work together with Web Modeler and Desktop modeler. 
IMO the intro should be 
1. What is each one good for
2. Why would you want to use them together
3. How do you use them together. 

This is not blocking

Reply:
What do you think about this changes? https://github.com/camunda/camunda-docs/pull/6508/commits/6b45f44f29061e377ecc121b600c88be57055a28